user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -188,10 +190,19 @@ private void createProteomicsGrid() {
     proteomicsMeasurementGrid.addClassName("measurement-grid");
     proteomicsMeasurementGrid.addColumn(
             proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value())
-        .setHeader("Measurement Code").setAutoWidth(true).setTooltipGenerator(proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value());
+        .setHeader("Measurement Code").setAutoWidth(true).setTooltipGenerator(
+            proteomicsMeasurement -> proteomicsMeasurement.measurementCode().value()).setFlexGrow(0);
     //Todo Should the sampleCodes be retrieved via a service or from column?
-    proteomicsMeasurementGrid.addColumn(proteomicsMeasurement -> proteomicsMeasurement.label().orElse("")).setHeader("Measurement Label").setTooltipGenerator(proteomicsMeasurement -> proteomicsMeasurement.label().orElse("")).setAutoWidth(true);
-    proteomicsMeasurementGrid.addColumn(proteomicsMeasurement -> proteomicsMeasurement.labelingType().orElse("")).setHeader("Measurement Label Type").setTooltipGenerator(proteomicsMeasurement -> proteomicsMeasurement.labelingType().orElse("")).setAutoWidth(true);
+    proteomicsMeasurementGrid.addColumn(
+            proteomicsMeasurement -> proteomicsMeasurement.label().orElse(""))
+        .setHeader("Measurement Label")
+        .setTooltipGenerator(proteomicsMeasurement -> proteomicsMeasurement.label().orElse(""))
+        .setAutoWidth(true).setFlexGrow(1);
```

Comment:
NIT: you can put `.setFlexGrow(1)` in another line so statements easily can be removed.

Reply:
I like the idea and adapted the call for all columns accordingly 👍